test/expected/build/library.ø/dest/resources/library/ø/rules/MyControl-dbg.support.js

```diff
@@ -0,0 +1,36 @@
+/*!
```

Comment:
As already mentioned above: We don't want to create -dbg files for *.support.js files. So they should just remain as-is.

Reply:
Done